resource-management/pkg/aggregrator/list-watch-impl.go

```diff
@@ -142,7 +142,11 @@ func (a *Aggregator) watchNodes(client SimInterface, crv types.TransitResourceVe
 				}
 
 				klog.V(9).Infof("Got node event from region manager, nodeId: %v", record.Node.Id)
-				a.processNode(&record)
+
+				// TODO: refine this go routine to sub functions
+				go func() {
```

Comment:
This is not good for RP down. Can you quantify how much it helps for daily case?